src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -256,6 +256,11 @@ def _get_attestation_endpoint(cloud_name):
     return attestation_endpoint_mapper.get(cloud_name, None)
 
 
+def _get_mhsm_dns_suffix(cloud_name):
+    mhsm_dns_suffix_mapper = {c.name: c.suffixes.mhsm_dns for c in HARD_CODED_CLOUD_LIST}
+    return mhsm_dns_suffix_mapper.get(cloud_name, None)
```

Comment:
for air-gapped cloud, cloud_name isn't in HARD_CODED_CLOUD_LIST, then mhsm_dns_suffix_mapper will be None, line 261 will throw NoneType error?

Reply:
line 261 will return None as the value.